packages/types/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@jessie.js/types",
+  "version": "0.1.0",
+  "description": "Description forthcoming.",
+  "keywords": [],
+  "author": "Agoric",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/endojs/Jessie/tree/master/packages/types#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/endojs/Jessie.git"
+  },
+  "bugs": {
+    "url": "https://github.com/endojs/Jessie/issues"
+  },
+  "type": "module",
+  "parsers": {
+    "js": "mjs"
+  },
+  "main": "./dist/types.cjs",
```

Comment:
I recently started trying to make some modules that have multiple public entry modules. For supporting before and after the `exports` feature, it starts to make sense to have shallow `index.js` and `index.t.js` files that *both* just `export { names } from './src/whatever.js'`, taking advantage of TypeScript’s ability to get types out of JSDocs, and then side by side with that module, similar files for each of the other named entry modules. This has the nice boon that `./src/whatever.js` can also export some things that are not package public API but present for unit tests.

Of course, this is all for naught because -rESM gets confused about whether those other entry modules are CJS or ESM.